scripts/build/Platform/Linux/destroy_cdk_applications.sh

```diff
@@ -61,6 +61,10 @@ then
     exit 1
 fi
 
+echo [cdk_installation] Install nodejs Node.js v17.x
```

Comment:
Question: why do we need to install nodejs again in destroy step?

And assume the reason we install nodejs as it is required for cdk? and version 17.x is recommended?

Reply:
Yes. nodejs is required by CDK and version 17.x is the latest version.

Reply:
cool, so why do you need to install it in destroy step?

Reply:
The scripts can be used to run automation tests manually as well on developers' local machine. Just in case someone wants to use this script to destroy testing CDK applications that were not deployed on the same machine and keep consistency with the Windows scripts.

Reply:
is `destroy` always being executed from the same environment as `deploy`? if not, then the dependencies needed to run CDK may not necessarily be present 🤔

Reply:
cool, so it never applies to automation process then, it is ok to keep it.